src/utilities/data-model.js

```diff
@@ -44,7 +44,7 @@ function savePhoto(geoHash, filePath) {
         console.log(err.message);
       });
 
-    var large = resizePhoto(filePath, 200, 200, 30)
+    var large = resizePhoto(filePath, 400, 500, 60)
```

Comment:
These numbers should be defined in some global configuration file and then imported here